jwql/tests/test_edb.py

```diff
@@ -15,16 +15,12 @@
 
     ::
 
-        pytest -s test_edb_interface.py
+        pytest -s test_edb.py
```

Comment:
I think this filename change should also be updated in `docs/source/test.rst`?

Reply:
yes, I missed that. thank you!